Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN yarn verify-format && yarn verify-spell
 
 FROM jenkins/jenkins:2.396-jdk11 as final
 USER root
+RUN ls /var/run && usermod -aG root jenkins
```

Comment:
I am adding `jenkins` user to `root` group because `docker` group does not exist. 

I cannot do something like `chmod 757 /var/run/docker.sock` during `docker build` because the file is mounted when the image is running.

Thoughts?